protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -484,6 +503,10 @@ definitions:
           If the stream is resumable or not. Should be set to true if the stream supports incremental. Defaults to false.
           Primarily used by the Platform in Full Refresh to determine if a Full Refresh stream should actually be treated as incremental within a job.
         type: boolean
+      is_file_based:
```

Comment:
nit: if we're representing files as something that can be included as part of standard records, maybe isntead of a whole stream being "file based" this is better expressed as "has files" or "has attachments"